docs/local-llm-operation-guide.md

```
@@ -0,0 +1,493 @@
+pip install pr-conflict-resolver
+
+# Verify installation
+pr-resolve --version
+```
+
+### Step 4: Configure Local LLM
+
+Create or update your configuration file:
+
+**config.yaml**:
+```yaml
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)